src/calendar/tests/unit/MonthPicker.ts

```diff
@@ -306,11 +307,11 @@ registerSuite({
 
```

Comment:
Could the `'Popup opens and closes on button click'()`, `'Popup closes with correct keys'()`, `'Arrow keys change year spinner'()`, and `'Clicking buttons changes year'()` use `sinon.stub` or `.spy`?